src/components/Nav/Nav.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+
+const Nav = () => {
+  return (
+    <nav>
+      <ul>
+        <li>
+          <NavLink to='/'>explore</NavLink>
+        </li>
+        <li>
+          <NavLink to='/map'>map</NavLink>
+        </li>
+        <li>
+          <NavLink to='/create'>create</NavLink>
+        </li>
+        <li>
+          <NavLink to='/profile'>profile</NavLink>
+        </li>
+      </ul>
+    </nav>
+  )
+}
+
```

Comment:
Great use of NavLink. We should be able to leverage the inherent "activeClassName" attr for this! I have also seen some "activeStyle" attr used that might be handy if we declare our preferred active styling within the component.